app/views/reservations/_date_filter.html.erb

```diff
@@ -0,0 +1,17 @@
+<div id="date_filter">
+  <h4>Filter By Date:</h4>
+  <p><%= filter_message(@time_counts, @all_counts, @filter, @view_all) %></p>
+  <%= form_tag update_index_dates_path, method: :put do %>
+    <%= hidden_field_tag "list[filter]", @filter %>
+    <%= label_tag :start_date, 'Begin' %>
+    <%= text_field "list", "start_date", class: "date_start_no_min span2", value: @start_date.strftime('%m/%d/%Y') %>
```

Comment:
Agreed, even though they'll be long for dates it will still fit with the rest of the column.